### PR TITLE
YQ-2549 Move yds integration tests to github 

### PR DESCRIPTION
**Pull request by kardymonds**
### Changelog entry 

...

### Changelog category 
* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/tests/fq/kikimr/test_recovery_match_recognize.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
```
Comment (kardymonds) [inline ydb/tests/fq/kikimr/test_recovery_match_recognize.py:1]: From https://a.yandex-team.ru/arcadia/kikimr/yq/tests/kikimr/test_recovery_match_recognize.py

File: ydb/tests/fq/kikimr/test_recovery_mz.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
```
Comment (kardymonds) [inline ydb/tests/fq/kikimr/test_recovery_mz.py:1]: From https://a.yandex-team.ru/arcadia/kikimr/yq/tests/kikimr/test_recovery_mz.py

File: ydb/tests/fq/kikimr/test_recovery.py [hunk 3]
```diff
@@ -0,0 +1,393 @@
+#!/usr/bin/env python
```
Comment (kardymonds) [inline ydb/tests/fq/kikimr/test_recovery.py:1]: From https://a.yandex-team.ru/arcadia/kikimr/yq/tests/kikimr/test_recovery.py

File: ydb/tests/fq/ya.make [hunk 4]
```diff
@@ -2,6 +2,7 @@ RECURSE_FOR_TESTS(
     common
     generic
     http_api
+    kikimr
```
Comment (uzhastik) [inline ydb/tests/fq/ya.make:5]: давайте как-нибудь поприятнее назовем папку

**Review by uzhastik — CHANGES_REQUESTED** [review]

File: ydb/tests/fq/ya.make (hunk 4 above)
Comment (kardymonds) [reply ydb/tests/fq/ya.make]: moved to yds/

**Review by uzhastik — APPROVED** [review]

**Pull request merged**